elearning-backend/app/Http/Controllers/API/AdminWordsController.php

```diff
@@ -40,6 +40,7 @@ public function getSingleWord($id)
         $word = Word::where('id', $id)->get()->first();
         $choices = json_decode($word->choices);
         $data = [
+            "id" => $word->id,
```

Comment:
i dont know why you reconstruct the data here but next time, since all data is related to word, there is no sense of reconstructing this data. you can just return the word variable to frontend.